### PR TITLE
Added products page

### PR DESCRIPTION
**Pull request by shivanigangadharan**
image

### CODE REVIEW

File: components.css [hunk 1]
```diff
@@ -1,9 +1,22 @@
...
 body{
     margin: 0;
+}
+.item-img{
+    width: 250px;
+    height: 300px;
+}
+.container-card{
+    width: 250px;
+    background: var(--white) !important;
```
Comment (AmishMishra11) [inline components.css:18]: Try not to use important

File: components.css [hunk 2]
```diff
@@ -1,9 +1,22 @@
...
 }
 body{
     margin: 0;
+}
+.item-img{
+    width: 250px;
+    height: 300px;
+}
+.container-card{
+    width: 250px;
```
Comment (AmishMishra11) [inline components.css]: use rems instead of px

File: components/filters.css [hunk 3]
```diff
@@ -0,0 +1,18 @@
+.container-filters{
+    background: var(--white);
+    position: fixed;
+    left: 0;
+    height: 100vh;
+    width: 250px;
```
Comment (AmishMishra11) [inline components/filters.css]: same as above use rems

**Review by AmishMishra11 — COMMENTED** [review]
image

your products are overlapping filters so you might want to look into it

File: components/filters.css (hunk 3 above)
Comment (shivanigangadharan) [reply components/filters.css]: done [0277212](https://github.com/shivanigangadharan/e-mart/pull/3/commits/0277212dc502ea55b39a7b6fef0c44f9933cd88f)

File: components.css (hunk 2 above)
Comment (shivanigangadharan) [reply components.css]: done [0277212](https://github.com/shivanigangadharan/e-mart/pull/3/commits/0277212dc502ea55b39a7b6fef0c44f9933cd88f)

File: components.css (hunk 1 above)
Comment (shivanigangadharan) [reply components.css:18]: I had to use that to over-write the styles coming from my component library style sheet. just to change background color here.

File: components/filters.css (hunk 3 above)
Comment (AmishMishra11) [reply components/filters.css]: great

File: components.css (hunk 2 above)
Comment (AmishMishra11) [reply components.css]: great

File: components.css (hunk 1 above)
Comment (AmishMishra11) [reply components.css:18]: ok then it alright

**Review by KanekarSnehal — APPROVED** [review]
No issues found code looks clean, you can merge it.

**Pull request merged**